lib/assets/index.md

```diff
@@ -115,9 +115,7 @@ if you prefer a more asynchronous approach. We also have volunteers on
 [StackOverflow](https://stackoverflow.com/tags/chef). If volunteer support isn't
 available, you can reach Chef Software's support team at [support@chef.io](mailto:support@chef.io).
 * **I Really Like IRC, What Do I Do?**
-  * The `#chef` IRC channel will be bridge to Slack for the foreseeable future, so
```

Comment:
Sigh, it's still on my todo list but job hunt is going to be top priority for a while. I just wouldn't mention the IRC bridge, it doesn't do the same thing and is also rarely stable enough to be worth using.